website/docs/r/notification_channel.html.markdown

```diff
@@ -181,6 +181,7 @@ resource "newrelic_notification_channel" "foo" {
 
   property {
     key = "service"
+    label = "Service Name" // Name of your service. If left out will be blank in UI.
```

Comment:
Unrelated, but could lower some complaints about it being blank in the future 😅

Reply:
I don't really understand what this comment means, so I am not really sure it would help customers...
Could elaborate what "service" is in this context? PD service? Also, what will be blank in the UI? Does it have to be the name of your PD services or would any name work?

Honestly, I am not really sure if this field affects UI at all: there's no separate UI for channels, and workflows do not rely on channel labels. Maybe the workflow builder UI does fetch channel name in the background, but I am not sure about this either...

Reply:
After speaking, write explicitly what this label is and why they should fill it.